app/app.controllers.js

```diff
@@ -23,12 +24,19 @@ exports.getAllTopics = ( req,res,next)=>{
     }).catch(next)
 }
 
-exports.getArticleById = (req, res, next) => {
-  const { article_id } = req.params;
-  selectArticlesById(article_id)
-    .then((article) => {
-      res.status(200).send(article);
-    })
-    .catch(next)
-    ;
-};
\ No newline at end of file
+
+exports.getArticle = (req, res, next) => {
```

Comment:
Would be good to rename getArticles (plural).  were getting all of them!

Reply:
Thanks Nick